lesson-7/todos/src/api.js

```diff
@@ -0,0 +1,43 @@
+import * as config from './config/api'
+
+export function getTodos() {
+    return fetch(`${config.BACKEND_URL}/todos`)
+        .then(result => {
+            return result.json();
+        });
+}
+
+export function addTodo(item) {
+    return fetch(`${config.BACKEND_URL}/todos/${item.uuid}`, {
+        method: 'POST',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(item)
+    }).then(result => {
+        return result.json();
+    });
+}
+
+export function updateTodo(item) {
+    return fetch(`${config.BACKEND_URL}/todos/${item.uuid}`, {
+        method: 'PUT',
+        headers: { 'Content-Type': 'application/json' },
+        body: JSON.stringify(item)
+    }).then(result => {
+        return result.json();
+    });
+}
+
+export function deleteTodo(uuid) {
+    return fetch(`${config.BACKEND_URL}/todos/${uuid}`, {
+        method: 'DELETE'
+    }).then(result => {
+            return result.json();
+        });
+}
+
+export function getSummaryHtml() {
+    return fetch(`${config.BACKEND_URL}/todoViews/summary`)
```

Comment:
This is not a good example - it kind of misses the point of SSR and actually makes things worse. Let's turn this into an usual JSON endpoint and either introduce proper SSR or drop the idea completely